specifications/KeyValueStore/Util.tla

```diff
@@ -0,0 +1,45 @@
+-------------------------------- MODULE Util --------------------------------
+EXTENDS Sequences, FiniteSets, Functions, Naturals, TLC
+\* Simple utility functions
+intersects(a, b) == (Cardinality(a \cap b) > 0)
```

Comment:
Defining this as "a \cap b # {}" would remove the dependence on FiniteSets.

Reply:
OK, I will try this.